blockstore/settings/base.py

```diff
@@ -48,7 +48,6 @@ def root(*x):
     'django_filters',
     'social_django',  # To let admin users log in using their LMS user account
     'waffle',
-    'corsheaders',
```

Comment:
Was added in https://github.com/edx/blockstore/pull/24 but has never been used in production.